RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate StatusView to Typescript

## Proposed changes

Migrate StatusView to Typescript

## Issue(s)	


## How to test or reproduce

- When logged in, click on the `hamburger menu`
- Then on the sidebar click on `Edit Status`, you should be moved to Status View, the view migrated.

## Screenshots

## Types of changes




## Checklist



## Further comments
